modules/RTC/JitsiLocalTrack.js

```diff
@@ -47,8 +52,9 @@ function JitsiLocalTrack(stream, track, mediaType, videoType, resolution,
 
     this.deviceId = deviceId;
     this.startMuted = false;
-    this.initialMSID = this.getMSID();
+    this.storedMSID = this.getMSID();
     this.inMuteOrUnmuteProgress = false;
+    this.peerConnections = [];
```

Comment:
According to the current switching logic between P2P and JVB it can not, but technically, as defined by the whole logic about add/remove, attach/detach and video mute it can.